06.bowling_object/frame.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require './shot'
+
+class Frame
+  attr_reader :shots, :idx
+
+  def initialize(shots, idx)
+    @shots = shots
+    @idx = idx
+  end
+
+  def calculate_frame(frames)
+    calculate_score + calculate_bonus(idx, frames)
```

Comment:
```suggestion
    calculate_score + calculate_bonus(frames)
```

`@idx`で参照できるので引数にしなくても大丈夫です